### ISSUE TITLE
HIT wont appear in catcher

### ISSUE THREAD

**Issue by GPiX1** [brandonhellman/mturk-suite#246]
Hello,

The link below when told to once or panda will NOT appear in the catcher window. The separate ID will however.

https://worker.mturk.com/projects/3TJJ8HSJVIX5LJVE1JI41HMTW6782G/tasks/accept_random

Any thoughts?

**Comment by WayotW** [comment]
I've had this issue from time to time with the hit catcher. Right now, I can't add https://worker.mturk.com/projects/3TCGLRK80JD69HKXMQ4V082QBK0ZQ6/tasks/accept_random to hit catcher. Adding the separate ID never worked for me either. Nothing works except fully reinstalling MTurk Suite. No clue what causes it.

**Comment by abc123meep** [comment]
Any updates on this? I'm having a similar issue. I added a HIT to the catcher, but then had to restart my browser for another reason. When i restored my open tabs, this specific HIT/watcher was gone. I then attempted to add it again but it wouldn't appear. I've tried manually adding the separate ID but that doesn't work either. 

For reference: https://worker.mturk.com/projects/3YVRJW0BX561A30OP5QFQRVMAE1Z4C/tasks/accept_random
